packages/apps-engine/src/definition/livechat/IPreLivechatRoomCreatePrevent.ts

```diff
@@ -0,0 +1,24 @@
+import { IRead, IHttp, IPersistence } from "../accessors";
+import { AppMethod } from "../metadata";
+import { ILivechatRoom } from "./ILivechatRoom";
+
+/**
+ * Handler called before a livechat room is created.
```

Comment:
```suggestion
 * Handler called before a livechat room is created.
 *
 * To prevent the room from being created, the app should throw an `AppsEngineException`
```